src/RiotPls.DataDragon/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace RiotPls.DataDragon.Extensions
+{
+    internal static class StringExtensions
+    {
+        public static string CapitalizeFirstLetter(this string value)
```

Comment:
will we keep this? we don't use it anymore

Reply:
We can get rid of it.